### ISSUE TITLE
Pretrained text models

### ISSUE THREAD

**Issue by natalypatti** [omerb01/TransBoost#1]
Hey!

I was reading the article of that work and the approach seems quite interesting :)

I would like to know if it is possible to apply the same method to pretrained models aimed at text classification, for example, BERT? Reading the work, I imagine that it is possible, but I would like to know your opinion if it would require major changes to the implemented code

**Comment by omerb01** [comment]
Hi @natalypatti !
It looks like it shouldn't be too hard.

In any case, as long as you can come up with the necessary 3 functions, which is where the creativity comes in :) , it is possible to apply TransBoost. In your case with text classification, it seems like you can simply use the softmax outputs from BERT exactly as specified in the paper.
